contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -44,11 +44,11 @@ contract EigenDABlobVerifier is IEigenDABlobVerifier {
      * @param blobHeader The blob header to verify
      * @param blobVerificationProof The blob verification proof to verify the blob against
      */
-    function verifyBlobV1(
+    function verifyDACertV1(
```

Comment:
Need to update the docstrings everywhere, which still mention "verifies the blob" instead of cert